EasyPost/Models/API/Batch.cs

```diff
@@ -89,7 +89,7 @@ public async Task<Batch> Buy()
         /// </summary>
         /// <param name="fileFormat">Format to generate the label in. Valid formats: "pdf", "zpl" and "epl2".</param>
         [CrudOperations.Update]
-        public async Task<Batch> GenerateLabel(string fileFormat = "pdf")
```

Comment:
Issue: This is a breaking change, we can't rip out a default here. 

What I meant was although sad and not consistent, let's not add it to the Shipment function. We could note this in code with a TODO to remove the default in an upcoming major release instead?

Reply:
Sounds good